### PR TITLE
[GPU] Merge kernel updates

### PR DESCRIPTION
**Pull request by jade-cho**
Fix a bug due to bias type mismatching
- convolution_gpu_bfyx_to_bfyx_f16

Refactoring
- depth_to_space_kernel_base
- depth_to_space_kernel_ref

Adjusting LWS
- eltwise_kernel_base

### CODE REVIEW

File: inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/mvn/mvn_kernel_b_fs_yx_fsv16_imad.cpp [hunk 1]
```diff
@@ -82,16 +82,33 @@ MVNKernelBase::DispatchData MVNKernel_b_fs_yx_fsv16_imad::SetDefault(const mvn_p
...
     auto activation_dt = GetActivationType(params);
+    jits.Merge(MakeTypeJitConstants(activation_dt, "ACTIVATION"));
     jits.Merge(MakeTypeJitConstants(activation_dt, "MEAN"));
+    jits.Merge(MakeTypeJitConstants(GetAccumulatorType(params), "ACCUMULATOR"));
     jits.AddConstant(MakeJitConstant("SIMD", simd));
     jits.AddConstant(MakeJitConstant("LWS", dispatchData.lws[0]));
     jits.AddConstant(MakeJitConstant("GWS", dispatchData.gws[0]));
     jits.AddConstant(MakeJitConstant("ITEM_GROUPS", dispatchData.itemsNum));
 
+    jits.AddConstant(MakeJitConstant("INPUT_SLICE_PITCH", 16));
```
Comment (isanghao) [inline inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/mvn/mvn_kernel_b_fs_yx_fsv16_imad.cpp:111]: I think mvn.cl is also updated for this to take effect?

File: inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/mvn/mvn_kernel_b_fs_yx_fsv16_imad.cpp [hunk 2]
```diff
@@ -82,16 +82,33 @@ MVNKernelBase::DispatchData MVNKernel_b_fs_yx_fsv16_imad::SetDefault(const mvn_p
...
+    const auto& input_dt = params.inputs[0].GetDType();
+
+    switch (input_dt) {
+        case Datatype::F32:
+        case Datatype::F16:
+            return Datatype::F32;
+        case Datatype::INT8:
+        case Datatype::UINT8:
+            return Datatype::INT32;
+        default: return Datatype::F32;
```
Comment (isanghao) [inline inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/mvn/mvn_kernel_b_fs_yx_fsv16_imad.cpp:95]: Isn't it necessary to evaluate performance impact from this change? Maybe we can make separate PR that may have potential performance impact..

File: inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/convolution_gpu_bfyx_to_bfyx_f16.cl [hunk 3]
```diff
@@ -89,7 +97,7 @@ KERNEL(convolution_bfyx_to_bfyx_f16)(
     bias_offset += split_idx * BIAS_LENGTH;
 #   endif
 
-    MAKE_VECTOR_TYPE(INPUT0_TYPE, OUTPUT_X_BLOCK_SIZE) dst = (MAKE_VECTOR_TYPE(INPUT0_TYPE, OUTPUT_X_BLOCK_SIZE))(DT_INPUT_BLOCK_READ(biases, bias_offset));
+    MAKE_VECTOR_TYPE(INPUT0_TYPE, OUTPUT_X_BLOCK_SIZE) dst = (MAKE_VECTOR_TYPE(INPUT0_TYPE, OUTPUT_X_BLOCK_SIZE))(DT_BIAS_BLOCK_READ(biases, bias_offset));
```
Comment (isanghao) [inline inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/convolution_gpu_bfyx_to_bfyx_f16.cl:100]: Could you describe more in detail of commit message? For example, what is changed and why it is changed.

Comment (jade-cho) [reply inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/convolution_gpu_bfyx_to_bfyx_f16.cl:100]: It's a bug due to bias data type mismatch. I am going to add details next update.

File: inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/mvn/mvn_kernel_b_fs_yx_fsv16_imad.cpp (hunk 1 above)
Comment (jade-cho) [reply inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/mvn/mvn_kernel_b_fs_yx_fsv16_imad.cpp:111]: I got it.

File: inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/mvn/mvn_kernel_b_fs_yx_fsv16_imad.cpp (hunk 2 above)
Comment (jade-cho) [reply inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/mvn/mvn_kernel_b_fs_yx_fsv16_imad.cpp:95]: An unit test for this already exists. I missed it.

File: inference-engine/src/cldnn_engine/cldnn_engine.cpp [hunk 4]
```diff
@@ -353,6 +354,7 @@ InferenceEngine::CNNNetwork clDNNEngine::CloneAndTransformNetwork(const Inferenc
             pass_config->disable<ngraph::pass::WeightsDequantizeToFakeQuantize>();
             pass_config->disable<ngraph::pass::SimplifyCTCGreedyDecoderSeqLen>();
             pass_config->enable<ngraph::pass::ConvertGather8ToGather7>();
+            pass_config->disable<ngraph::pass::ConvertMinimum>();
```
[the remark this answers is not in the document]
Comment (jade-cho) [reply inference-engine/src/cldnn_engine/cldnn_engine.cpp:357]: I will remove this file at this PR.

File: inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/convolution_gpu_bfyx_to_bfyx_f16.cl (hunk 3 above)
Comment (jade-cho) [reply inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/convolution_gpu_bfyx_to_bfyx_f16.cl:100]: It is a bug due to bias data type mismatching and might still affect to openvino/master. I will describe more in commit message.

**Review by isanghao — APPROVED** [review]
